source/model/database.py

```diff
@@ -0,0 +1,44 @@
+from commands.keys import *
+import os, json
+from model.table import *
+from output.exceptions import *
+
+class Database:
+    def __init__(self):
```

Comment:
database should have a `self.path = DATABASE.PATH + self.database_name`

DATABASE.PATH // Should be a constant in DATABASE class